index.html

```diff
@@ -35,7 +37,174 @@ <h1>I'm Safar Ali <br> A thriving <br> Software Developer </h1>
                 </ul>
             </div>
         </section>
-    </section>    
+    </section>
+
+    <!-- Main-section -->
+
+    <main class="main">
+
+        <!-- page-2 -->
+
+        <section class="page-2">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-1.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Tonic</h1>
+            </div>
+            <div class="year">
+                <p>Safar</p>
+                <p>Backend Dev</p>
+                <p>2015</p>
+            </div>
+            <div class="para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="skills">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
+            </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them is a `li`

```
<ul>
  <li>..</li>
  <li>..</li>
  <li>..</li>
</ul>
```

Reply:
@Hamzaoutdoors ,

Can you please tell me if I keep it this way? Because It hasn't been mentioned in the requirements.

Reply:
@Hamzaoutdoors ,

Hello brother. Can you please reply here?
I am waiting for your answer.

Thanks in advance.

Reply:
> * [ ]  The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them is a `li`
> 
> ```
> <ul>
>   <li>..</li>
>   <li>..</li>
>   <li>..</li>
> </ul>
> ```

- [ ] Please consider making changes required by the previous review, it is required for all technologies to use ul tag instead, kindly use padding, margin font-size, font-family, and the same color as it is provided in the Figma template, you design should stick to the original one.
- [ ] Your design
<img width="317" alt="y111" src="https://user-images.githubusercontent.com/79058364/153719650-ab8dc270-c565-41a2-97e9-ed903e34e8b7.png">

- [ ] Make these changes on all of your cards to match the Figma design below, please.

- [ ] Figma
<img width="256" alt="f111" src="https://user-images.githubusercontent.com/79058364/153719708-39fe5d3f-5b5b-48d8-9d8c-4793b1d7c155.png">

Reply:
Hey @safar1212 

As you can see in the screenshot below you were asked to list technologies for each project card section using `ul` and `li`

![Capture](https://user-images.githubusercontent.com/80895497/153719829-835cb8c4-e0c1-4e92-9834-71c50de9ddf2.PNG)